components/alerts.css

```diff
@@ -0,0 +1,78 @@
+.alert{
+    padding: 1.3rem;
+   background-color: var(--white-color);
+   border-radius: 7px;
+   display: flex;
+   align-items: center;
+   justify-content: space-between;
+   flex-wrap: wrap;
+   gap: 1rem;
+    max-width: 380px;
+    min-height: 5.5rem;
+    font-family: "Poppins",sans-serif;
+ }
+ .alert button{
+   background-color: transparent;
+   font-size: 1.8rem;
+     color: var(--alert-para-color);
+     cursor: pointer;
+     line-height: 0;
+     border: none;
+     /* width: 1.5rem;
```

Comment:
Remove the commented code. Everything else looks good

Reply:
done